cli/src/command/generate_template.rs

```diff
@@ -34,17 +35,25 @@ pub struct GenerateTemplate {
     /// Number of validators in the network.
     #[structopt(long)]
     pub validators_count: u32,
+    /// Supervisor service mode. Could be 'simple' or 'decentralized'.
+    ///
+    /// Default: simple.
+    #[structopt(long)]
+    pub supervisor_mode: Option<SupervisorMode>,
 }
 
 impl ExonumCommand for GenerateTemplate {
     fn execute(self) -> Result<StandardResult, Error> {
-        let config_template = CommonConfigTemplate {
+        let supervisor_mode = self.supervisor_mode.unwrap_or(SupervisorMode::Simple);
```

Comment:
I don't like that default value is hidden deep in the sources. It's not as explicit as it was before.